velox/common/memory/MappedMemory.h

```diff
@@ -218,6 +302,14 @@ class MappedMemory : public std::enable_shared_from_this<MappedMemory> {
     uint64_t totalInSizeClasses;
     // Total in standalone large allocations via allocateContiguous().
     uint64_t totalLarge;
+
+    AllocateBytesCounters operator-(const AllocateBytesCounters other) const {
```

Comment:
The name of this stuck is inconsistent with other similar structs (SizeStats and Stats). Would be nice to unify.